workers/unity/Assets/Gdk/TestUtils/Tests/Editmode/HybridGdkSystemTests.cs

```diff
@@ -0,0 +1,115 @@
+using System;
+using JetBrains.Annotations;
+using NUnit.Framework;
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.TestUtils.EditmodeTests
```

Comment:
Presumably in playmode tests we can just add the systems to a world and let it do it's thing?

Reply:
I haven't considered playmode tests, I am guessing if you have the bootstrap as part of the world it should be OK